README.md

```diff
@@ -317,24 +317,54 @@ Deploy [web validate](https://github.com/retaildevcrews/webvalidate) to drive co
 
 ```bash
 
+# Add Log Analytics extension
+az extension add -n log-analytics
+
+# create Log Analytics for the webv clients
+az monitor log-analytics workspace create -g $He_App_RG -l $He_Location -n $He_Name -o table
+
+# retrieve the Log Analytics ID
+export He_LogAnalyticsId=$(az monitor log-analytics workspace show -g $He_App_RG -n $He_Name --query customerId -o tsv)
```

Comment:
the env var should hold the command for the value, not the value itself (security issue). then you $(exec $He_LogAnalyticsId) to retrieve the value real-time.